docs/submit-transactions.md

```diff
@@ -16,13 +16,13 @@ and behaviour of each one of them.
 
 ⓘ All the transactions provide the same [Command-Line Options](#command-line-options).
 
-- Transfer funds from an account to another:
+- Transfer funds from an account to another account, possibly org's:
```

Comment:
```suggestion
- Transfer funds from the author's account to another account, which can be a local, org's, or user's account.
```

Reply:
A bit too many "accounts' though.

Reply:
That's the only word we have :smile: Fixed.